platform/security/security-migratable/src/main/java/org/codice/ddf/security/migratable/impl/SecurityMigratable.java

```diff
@@ -38,10 +40,13 @@
    *
    * <p>1.0 - initial version
    */
-  private static final String CURRENT_VERSION = "1.0";
+  private static final String CURRENT_VERSION = "2.0";
```

Comment:
❗️ Can we document above what has changed in version 2.0?

Reply:
Reverted this change so that the migratable versions are remaining the same.